skiko/build.gradle.kts

```diff
@@ -145,10 +145,12 @@ fun compileNativeBridgesTask(os: OS, arch: Arch): TaskProvider<CompileSkikoCppTa
                 val iosArchFlags = when (arch) {
                     Arch.Arm64 -> arrayOf(
                         "-target", "arm64-apple-ios",
+                        "-mios-version-min=11.0",
```

Comment:
Is it added because this is the minimal version supported by Skia?

Reply:
Yes, the new version of skia uses some language features that are allowed only from ios11